src/pages/index.vue

```diff
@@ -106,10 +106,23 @@ export default {
     WatchingIcon,
   },
   computed: {
-    ...mapGetters(['getUserId', 'getUserIsCivicLiker']),
+    ...mapGetters(['getUserId', 'getUserIsCivicLiker', 'getUserBookmarks']),
+  },
+  mounted() {
+    this.fetchSharedContent();
   },
   methods: {
     getOAuthLoginAPI,
+    ...mapActions(['refreshBookmarkList']),
+    async fetchSharedContent() {
+      try {
+        const promises = [];
+        if (!this.getUserBookmarks.length) promises.push(this.refreshBookmarkList());
+        await Promise.all(promises);
+      } catch (err) {
+        console.error(err); // eslint-disable-line no-console
+      }
+    },
```

Comment:
For prefetching?

Reply:
It depends if a content card needs to know if its bookmarked or not, in this PR's case the `Add bookmark` or `Remove bookmark` text needs this info across `/index`